guide.md

```diff
@@ -100,52 +168,40 @@ across that the tools we show are general:
 - Sphinx is not only for Python
 - Read the Docs is not only for Github
 
-In the fourth exercise, we first test the availability of Sphinx. This test
-needs to be done in a terminal where python is available. It can take a bit of
+In the Sphinx exercise, we first test the availability of Sphinx. This test
+needs to be done in a terminal where Python is available. It can take a bit of
 time to get everybody on board with Sphinx.  It can be nice to make a short
 break if you need more time to fix problems.
```

Comment:
What if each lesson had a "getting started verification", that could be done as homework the night before to ensure that you could get started immediately?  Then, for days 2 and 3, we can say "do this to make sure you are ready for tomorrow" (or equally, do them all in advance).

Reply:
I think this would be helpful. We need to make sure to not repeat/diverge wrt installation. What would be a good place to track this issue/idea?